### PR TITLE
Added some additional keywords to mosaic

### PR DESCRIPTION
**Pull request by amcody**
This adds keywords requested by MAST, such as FILTER='Kepler', MJD-BEG and END, and XPOSURE.

### CODE REVIEW

File: k2mosaic/mosaic.py [hunk 1]
```diff
@@ -171,6 +182,12 @@ def _make_primary_hdu(self):
         hdu.header.cards['DATE-OBS'].comment = 'TSTART as UTC calendar date'
         hdu.header['DATE-END'] = self.dateend
         hdu.header.cards['DATE-END'].comment = 'TSTOP as UTC calendar date'
+        hdu.header['MJD-BEG'] = self.mjdbeg
+        hdu.header.cards['MJD-BEG'].comment = 'TSTART as modified barycentric Julian date'
+        hdu.header['MJD-end'] = self.mjdend
```
Comment (barentsen) [inline k2mosaic/mosaic.py]: This probably doesn't change the output, let's use uppercase here:
```suggestion
        hdu.header['MJD-END'] = self.mjdend
```

File: k2mosaic/mosaic.py [hunk 2]
```diff
@@ -183,6 +200,8 @@ def _make_primary_hdu(self):
         hdu.header.cards['TELESCOP'].comment = 'telescope'
         hdu.header['INSTRUME'] = 'Kepler Photometer'
         hdu.header.cards['INSTRUME'].comment = 'detector type'
+        hdu.header['FILTER'] = 'Kepler'
+        hdu.header.cards['FILTER'].comment = 'filter'
```
Comment (barentsen) [inline k2mosaic/mosaic.py]: I guess we can be a bit more informative in this comment, I suggest:
```suggestion
        hdu.header.cards['FILTER'].comment = 'photometric passband'
```

File: k2mosaic/mosaic.py [hunk 3]
```diff
@@ -254,7 +275,13 @@ def _make_image_extension(self, extname, data):
         hdu.header['TELAPSE'] = frametim/3600./24. * num_frm
         hdu.header.cards['TELAPSE'].comment = '[d] TSTOP - TSTART'
 
-        for keyword in ['EXPOSURE', 'LIVETIME', 'DEADC', 'TIMEPIXR', 'TIERRELA',
+        hdu.header['EXPOSURE'] = int_time/3600./24. * num_frm
+        hdu.header.cards['EXPOSURE'].comment = '[d] time on source'
+
+        hdu.header['LIVETIME'] = frametim/3600./24. * num_frm * deadc
```
Comment (barentsen) [inline k2mosaic/mosaic.py]: This probably doesn't change anything, but it's slightly better to not repeat the computation, i.e.:
```suggestion
        hdu.header['LIVETIME'] = hdu.header['TELAPSE'] * deadc
```

**Comment by barentsen** [comment]
@amcody This all looks good!  I nit-picked a few small things, but this will be good to merge!

**Comment by amcody** [comment]
Should be fixed now, @barentsen!

**Comment by barentsen** [comment]
Thanks!

**Pull request merged**